.github/workflows/pipeline-deploy-bootstrap.yaml

```diff
@@ -12,10 +12,10 @@ jobs:
       with:
         ENV: dev
       secrets: inherit
-    deploy-bootstrap-dos-test:
-      name: Run bootstrap for dos test account
-      uses: ./.github/workflows/deploy-bootstrap.yaml
-      with:
-        ENV: test
-      secrets: inherit
+    # deploy-bootstrap-dos-test:
```

Comment:
Maybe remove the dead code (the sonarqube checks will eventually complain about it anyway)